cli/src/awsbatch/awsbsub.py

```diff
@@ -560,7 +560,7 @@ def main():
             retry_attempts=args.retry_attempts,
             timeout=args.timeout,
             env=[
-                ("MASTER_IP", config.head_node_ip),  # TODO remove
+                ("PCLUSTER_HEAD_NODE_IP", config.head_node_ip),  # TODO remove
```

Comment:
We are getting head node ip internally and we kept this to avoid breaking existing clusters. But this patch has been done 2 years ago so we can get rid of this. See fcc75f44974870452dfce63dd36ca3d106cb5d5c